### ISSUE TITLE
Add 'Delete file' and 'Add another' buttons to  Figma file form review section

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/vets-design-system-documentation#417]
# Feature Request

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## Is this feature request relating to an existing component or utility? Please describe.

- Component/utility name: We have a review form template in the Sketch file. We should add the 'Delete file' and 'Add another' buttons to these templates so designers have a visual to refer to.

## Additional context
<img width="585" alt="Screen Shot 2021-03-30 at 10 30 42 AM" src="https://user-images.githubusercontent.com/72398073/113186120-41c7f200-9225-11eb-9720-1beebab6dca1.png">
<img width="580" alt="Screen Shot 2021-03-30 at 11 29 41 AM" src="https://user-images.githubusercontent.com/72398073/113186174-50aea480-9225-11eb-8e2f-56f835970389.png">

**Comment by caw310** [comment]
@CrystabelReiter , let's try to fit this in when you have time.

**Comment by humancompanion-usds** [comment]
@allison0034 - This issue is admittedly quite old, but I'll try to address it. This is what we have in Sketch: https://sketch.com/s/c8df169f-5b02-4999-befb-34c7b3b62ba9 

This [pattern](https://design.va.gov/patterns/ask-users-for/service-history) doesn't cover uploading an attachment and thus neither does the pattern in Sketch. But I see in the screenshot that it is a valid variation. Do you know where we are doing this so I can go have a look?

**Comment by allison0034** [comment]
I confirmed the "Add another" is now in the review page for the list and loop. 
<img width="586" alt="Screen Shot 2023-10-12 at 1 07 20 PM" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/72398073/a669b445-ef55-4524-a94c-f5ced924a026">

**Comment by allison0034** [comment]
Same for the delete file. I checked the 21-526EZ.
<img width="578" alt="Screen Shot 2023-10-12 at 1 10 52 PM" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/72398073/874dcb76-7e12-4ecd-a3c0-6ecdffb7405e">

**Comment by caw310** [comment]
@danbrady @LWWright7 is this fixed in Figma? If so let's close this issue.

**Comment by LWWright7** [comment]
@caw310 This can be closed after I complete [this ticket](https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/2460), during this sprint.  Right @danbrady ??

**Comment by danbrady** [comment]
Yes, I believe so. However, if any steps have a file upload we'll need to circle back to it once the updated File Input component (with the drop zone and optional card) is created. If you see that in any of the patterns, please add it to an issue so we can track it. Thanks!

**Comment by LWWright7** [comment]
@danbrady  Will do! Thanks!

**Comment by caw310** [comment]
> @caw310 This can be closed after I complete [this ticket](https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/2460), during this sprint. Right @danbrady ??

Yes you can close this onc eyou're done with the other ticket. Thanks.

**Comment by humancompanion-usds** [comment]
I don't think this is done. What this originally asked for was showing how we handle list and loop patterns, like Service History, in the Review page in a form. We could do that in one of the following ways:

1. Add a template for the [Help users to check answers] pattern which shows the Review template in various states and include list and loop as one of those. 
2. Do the same as above but instead just add on to the current Review page template annotated example.
3. Add an example in Service History that depicts how that shows up on the Review page.

I think #2 or #1 are the way to go.

**Comment by humancompanion-usds** [comment]
@danbrady - Can you review this issue and see if this has been handled and if not get it slated for an upcoming sprint with @caw310 ? Thanks.

**Comment by caw310** [comment]
Dan reviewed this and said that it doesn't look done. I will put this in a future sprint so we can get this done.

**Comment by danbrady** [comment]
The Multiple Responses pattern was rewritten recently with new examples. I believe the original request is no longer applicable.
